client/src/components/VersionSaveComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Input } from 'antd'
+import React, { useState } from 'react'
+import { GetAnswerDocument, useSaveVersionMutation } from '../generated/graphql'
+import { messageService } from '../services/message'
+
+const VersionSaveComponent: React.FC<{
+  answerId: string
+  commitMessage: string
+  versionId: string
+}> = props => {
+  const [inputState, setInputState] = useState('')
+
+  const [saveVersion] = useSaveVersionMutation({
+    onCompleted: ({ saveVersion }) => {
+      if (saveVersion === true) {
+        messageService.success('New version saved')
+      } else {
+        messageService.error('No changes detected, not saving')
+      }
+    },
+    variables: {
+      id: props.answerId,
+      commitMessage: inputState
+    },
+    refetchQueries: [
+      {
+        query: GetAnswerDocument,
+        variables: {
+          id: props.answerId
+        }
+      }
+    ]
+  })
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
+      <Input
```

Comment:
As a user I would expect the save-action to also be triggered when I press `Enter` in the input (see `onPressEnter` callback for `Input` for example)